### PR TITLE
migrate jekyll metadata to JBook

### LINKED ISSUE #50: remove/migrate Jekyll styler metadata from page headers
Most of the pages have Jekyll style metadata like this at the top of each page:

```markdown
---
layout: page
title: Childcare
---

# Childcare

**Tags:** <span style="color:red">Parents and Carers</span>, <span style="color:red">Participant Diversity</span>, <span style="color:red">Scholarships</span>

✅ Offer financial support for childcare.
 - Build it into your budget. 
   - Link it to sponsorships, or university partners, ...
```

Jbook uses different metadata, need to research more, but it's built on top of Sphinx, so is similar. May not need them at all because jbook automatically creates table of contents and sections from markdown headers like `# Childcare` automatically.

### PR DESCRIPTION
**Pull request by mikofski**
* closes #50 
* add JBook section labels
* update links using Jekyll permalinks to use JBook `{ref}` and `{doc`} roles or markdown as needed

### CODE REVIEW

**Comment by mikofski** [comment]
Hi @noatamir & @terezaif I think this is ready. I replaced all of the Jekyll front matter with JBook labels to the top level section. TBH, I couldn't find a single instance where they were necessary, since most of the pages are referenced using the `{doc}` directive or Markdown directly. I fixed one broken link in Ch. 12 - Assessment to the Event Registration page. And I also fixed a few typos & spelling mistakes when I saw them flagged on my screen.

PS I had to empty-cache & reload the PR review b/c it always uses the same URL, and my browser didn't reload it automatically.

File: DISCOVER/13_taxonomy.md [hunk 1]
```diff
@@ -1,17 +1,13 @@
----
-layout: page
-title: Taxonomy
----
-
+(topics)=
 # Taxonomy
```
Comment (terezaif) [inline DISCOVER/13_taxonomy.md:2]:
```suggestion
# Glossary
```

**Review by terezaif — APPROVED** [review]
Screenshot 2023-02-07 at 11 57 18
Based on this conversation.. changed also Taxonomy to Glossary.. if you feel it should be it's own pr then pls ignore

File: DISCOVER/13_taxonomy.md (hunk 1 above)
Comment (mikofski) [reply DISCOVER/13_taxonomy.md:2]: I like Glossary too. There is a built in feature for Glossary in JBook / Sphinx. I think we can repurpose this page. Also the way the current taxonomy page is used is to collect the tags and redirect them. With #60 this automatically gets generated with Melissa's sphinx-tags tool, and we can name it anything we want, but by default it's named "Tags Overview"

**Pull request merged**